src/main/java/yasp/Main.java

```diff
@@ -2,35 +2,61 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.FileInputStream;
 import java.io.OutputStream;
 import java.net.InetSocketAddress;
 import com.sun.net.httpserver.HttpExchange;
 import com.sun.net.httpserver.HttpHandler;
 import com.sun.net.httpserver.HttpServer;
     
 public class Main {
+    
+    public static void main(String[] args) {
+	try { 
+                       
+	    if (args.length > 0 && args[0].equals("--file")) { System.exit(parse(args[1])); }
+            else if (args.length >0 && args[0].equals("--")) { System.exit(parseStream(System.in, System.out)); }
+            else { startServer(args); }
+	}
+	catch (Exception e) {
+	    e.printStackTrace();
```

Comment:
do you need to catch the exception here?  If we have one, we should crash since the startup is incorrect.